src/hotspot/share/runtime/globals.hpp

```diff
@@ -1994,6 +1994,19 @@ const int ObjectAlignmentInBytes = 8;
                 "Unconditionally record nmethod dependencies on class "     \
                 "rewriting/transformation independently of the JVMTI "      \
                 "can_{retransform/redefine}_classes capabilities.")         \
+                                                                            \
+  product(bool, UseSecondarySuperCache, true, DIAGNOSTIC,                   \
+          "Use secondary super cache during subtype checks")                \
```

Comment:
Is it guard for current implementation or both?

Reply:
Just the current.